### PR TITLE
[GEOS-10025] docs: Improve the translation

### PR DESCRIPTION
**Pull request by hellorayza**
[GEOS-10025](https://osgeo-org.atlassian.net/browse/GEOS-10025)

  

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [ ] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [x] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [ ] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [ ] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [ ] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

# Supplement

had run  `mvn clean -f doc/zhCN install -X` and checked target .

image

### CODE REVIEW

File: doc/zhCN/user/source/styling/css/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+         </FeatureTypeStyle>
+        </UserStyle>
+        </NamedLayer>
+    </StyledLayerDescriptor>
+
+现在，让我们从完成相同任务的 CSS 文件开始。
+
+首先，进入样式页面，点击 :guilabel:`add a new style` 链接来开始新样式。在“新样式”页面中，执行以下操作::
+
+* 将新样式命名为任何您喜欢的名称，例如 ``csstutorial``
```
Comment (aaime) [inline doc/zhCN/user/source/styling/css/tutorial.rst:125]: One of the screenshots says "csstorial" though.

Comment (hellorayza) [reply doc/zhCN/user/source/styling/css/tutorial.rst:125]: The screenshot has been updated.

**Review by mprins — CHANGES_REQUESTED** [review]
`doc/zhCN/user/source/styling/css/images/tutorial_noborder.png` still has "csstorial" in the heading

**Comment by hellorayza** [comment]
> `doc/zhCN/user/source/styling/css/images/tutorial_noborder.png` still has "csstorial" in the heading

have been corrected

**Review by mprins — APPROVED** [review]

**Comment by mprins** [comment]
Not sure if this requires a CLI / CLA, most files are copied images from the EN docs

**Comment by aaime** [comment]
@mprins I don't think so, we don't need to donate documentation to other projects like we occasionally do for code. Merging.

**Pull request merged**